src/avfoundation.cs

```diff
@@ -10389,6 +10643,10 @@ interface AVPlayerItem : NSCopying {
 		[Export ("preferredPeakBitRate")]
 		double PreferredPeakBitRate { get; set; }
 
+		[iOS (11, 0)]
```

Comment:
since only iOS headers were updated you need to add `[NoMac]` (XM intro tests are failing) `[NoTV]` and `[NoWatch]` attributes.

Reply:
Correct, Sorry for thatI forgot theNoMac and NoTV, watch is already ignored since is in the top of the class.